benchmarks/stemming_sample.txt

```diff
@@ -6101,7 +6101,7 @@
 ΕΤΣΙ,ΕΤΣΙ
 ΕΤΩΝ,ΕΤ
 ΕΥ,ΕΥ
-ΕΥΑ,ΕΥ
+ΕΥΑ,Ε
```

Comment:
This one's a bit weird imho - not sure whether we should allow the stem to be 1 char.